documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -21,7 +21,7 @@ toc::[]
 
 [NOTE]
```

Comment:
While at it, let's remove this entire note. The new capturing implementation has matured for one year, and we are going to drop it in 2.0. So we can remove the hint at the old one from the docs in 1.9 (it still will be available if someone really needs it).

Reply:
Done.